ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -571,4 +571,19 @@ FAN_SS_AL:
   - control_mode
   - run_mode
   - schedule_run_command
-  - control_status
\ No newline at end of file
+  - control_status
+
+FAN_SS_SC_FA_SR:
```

Comment:
is this not just a `FAN_SS_VSC`? the field `exhaust_fan_speed_percentage_sensor` should simply be `speed_percentage_command`, since it is obvious from context that it is for an exhaust fan, if the entity name is `EF-<number>`.

Reply:
FAN_SS_AL is nearer for me sir. will do as you note for speed_percentage_command. Thanks sir